src/main/kotlin/com/example/ajaxproject/exeption/GlobalExceptionHandler.kt

```diff
@@ -13,10 +13,14 @@ class GlobalExceptionHandler {
         return ResponseEntity.status(HttpStatus.NOT_FOUND).body(ex.message)
     }
 
-
-    @ExceptionHandler(UserNotFoundException::class)
-    fun handleUserNotFoundException(ex: UserNotFoundException): ResponseEntity<Any> {
+    @ExceptionHandler(NotFoundException::class)
+    fun handleUserNotFoundException(ex: NotFoundException): ResponseEntity<Any> {
         return ResponseEntity.status(HttpStatus.NOT_FOUND).body(ex.message)
     }
 
+    @ExceptionHandler(WrongActionException::class)
+    fun handleUserNotFoundException(ex: WrongActionException): ResponseEntity<Any> {
+        return ResponseEntity.status(HttpStatus.LOCKED).body(ex.message)
+    }
+
```

Comment:
before last brace should be no empty lines

Reply:
fixed it in all files